src/services/newValidation.js

```
@@ -8,43 +8,53 @@ export async function validateUser(
 ) {
   let user = {};
 
-  if (authTypes.includes("ID")) {
-    if (userType == "student") {
-      user["isUserIdValid"] = await userAPI.verifyStudent({
-        student_id: userInformation["student_id"],
-        auth_group_id: authGroupId,
-      });
-    }
-    if (userType == "teacher") {
-      user["isUserIdValid"] = await userAPI.verifyTeacher({
-        teacher_id: userInformation["teacher_id"],
-      });
-    }
-  }
+  let studentVerificationParams = {
```

Will `userInformation` always contain `student_id`?

---

not always. as of now returning `undefined` for non student logins . .silently failing. will change code